sinfonier-backend-api/config/Routes.py

```diff
@@ -18,6 +18,7 @@ def __init__(self):
         self.api.add_route('/topologies/{id}/launch', LaunchTopology.LaunchTopology())
         self.api.add_route('/topologies/{id}/stop', StopTopology.StopTopology())
         self.api.add_route('/topologies/{id}/log', GetTopologyLog.GetTopologyLog())
+        self.api.add_route('/topologies/{id}/logsizes', GetTopologyLogSizes.GetTopologyLogSizes())
```

Comment:
Lets merge 1.0.1 with 1.1.0 first to get test code. Later I will make a new PR with tests